assets/styles/landing.scss

```diff
@@ -0,0 +1,33 @@
+.body {
+    border-top: 8px solid $cream-can;
+    font-family: 'Open Sans';
```

Comment:
add a default font in case the Open Sans font doesn't load. it should look like this:
```
font-family: 'Open Sans', sans-serif;
```